### PR TITLE
Various things adjusted

### PR DESCRIPTION
**Pull request by apeltzer**
* Citations added
* More bioinfx
* Reworded bits

... lots more to come! :-)

### CODE REVIEW

File: content/03.results-and-discussion.md [hunk 1]
```diff
@@ -103,34 +77,25 @@ related taxa - a common challenge in ancient bacteria genome reconstruction
...
+calculator (endorS.py, by A.A.V), to provide an approximate level of
 on-target DNA yield within a sample. Secondly, ancient metagenomic studies may
 include comparative samples from living day individuals
 (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data
 privacy, nf-core/eager includes a 'strip_fastq' script (by M.B.) which creates
 raw FASTQ files but with reference-genome mapped reads removed. This then
 allows safe upload of sequencing data to public repositories with identifiable
 data removed. When using a human reference genome, nf-core/eager also can give
-estimates of the biological sex of a given individual with Sex.DetEERRmine
+estimates of the biological sex of a given individual with `Sex.DetEERRmine`
```
Comment (jfy133) [inline content/03.results-and-discussion.md]: Do we want to do code blocks for all tools? I suspect 99.99% of journals won't allow formatting of this.

File: content/03.results-and-discussion.md [hunk 2]
```diff
@@ -147,16 +112,6 @@ by providing practical and 'applied' knowledge to how aDNA characteristics
...
-in a prominent position within the code, rather than custom java code and
-classes. This is particularly relevant for researchers ancient DNA, who often
-come from social sciences and do not have backgrounds or training in
-computational biology. We hope this will motivate further bug fixes and feature
-contributions to keep the updated pipeline with standard practises during a
-longer life-cycle of this version of the pipeline. This will also be supported
-by the active and welcoming nf-core community who provide general guidance and
-advice on developing nextflow and nf-core pipelines.
+also improve open-source community contributions to the pipeline. While Nextflow
+is written primarily in Groovy, the Nextflow DSL simplifies a number of concepts to an intermediate level that bioinformaticians without Java/Groovy experience can easily access (regardless of own programming language experience). Furthermore, Nextflow places ubiquitous command-line interfaces, such as bash,
```
Comment (jfy133) [inline content/03.results-and-discussion.md]:
```suggestion
is written primarily in Groovy, the Nextflow DSL simplifies a number of concepts to an intermediate level that bioinformaticians without Java/Groovy experience can easily access (regardless of own programming language experience). Furthermore, Nextflow places ubiquitous and more widely known command-line interfaces, such as bash,
```

File: content/04-conclusion.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
 ## Conclusion
 
 nf-core/eager is an efficient, portable, and accessible pipeline for processing
-ancient DNA genomic data. The re-implementation with nextflow and nf-core
+ancient DNA genomic data. The re-implementation with Nextflow and nf-core
```
Comment (jfy133) [inline content/04-conclusion.md:4]: Are we going upper case o lower case? Above you put it lower case?

File: content/05.methods.md [hunk 4]
```diff
@@ -2,30 +2,43 @@
...
 
-Example data is included.
+#### Profiles
 
-Output includes both files for close inspection and downstream analysis, as
-well as the emailable MultiQC report.
+We utilize a central configuration profile repository to enable users from various institutions to use pipelines on their particular infrastructure [more easily](https://github.com/nf-core/configs). For details on how to add your own profile to that repository, there are multiple resources listed in this repository to add your own configuration profile with help from the nf-core community.
 
-## Competing Interests
+Users can customize this infrastructure profile once either alone or with the nf-core community, or with their local IT to make sure that the pipeline runs successfully and can then rely on the Nextflow and nf-core framework to ensure compatibility upon further infrastructure changes. In order to run the nf-core/eager pipeline at the Max Planck Institute for the science of human history in Jena, users only have to run:
```
Comment (jfy133) [inline content/05.methods.md]:
```suggestion
Users can customize this infrastructure profile once by themselves, with the nf-core community, or with their local system administrator to make sure that the pipeline runs successfully and can then rely on the Nextflow and nf-core framework to ensure compatibility upon further infrastructure changes. For example, in order to run the nf-core/eager pipeline at the Max Planck Institute for the Science of Human History in Jena, users only have to run:
```

File: content/05.methods.md [hunk 5]
```diff
@@ -2,30 +2,43 @@
...
 
-No competing interests are declared.
+```bash
+nextflow run nf-core/eager -r 2.1.0 -profile shh_cdag,test
+```
+
+This runs the testing profile of the nf-core/eager pipeline on the HPC system at the MPI SHH in Jena/Germany. Similar configs for multiple institutions already exist, so users might not even have to write their own once.
```
Comment (jfy133) [inline content/05.methods.md]:
```suggestion
This runs the testing profile of the nf-core/eager pipeline with parameters specifically adapted to the HPC system at the MPI-SHH. In some cases, similar institutional configs for other institutions mat already exist (originally utilised for different nf-core pipelines), so users need not write their own.
```

File: content/05.methods.md [hunk 6]
```diff
@@ -2,30 +2,43 @@
...
+
+#### Inputs
+
+The pipeline can be started using (raw) FastQ from sequencing or pre-mapped BAM files. Additionally, the pipeline requires a FASTA reference genome. If users have complex setups, e.g. multiple sequencing lanes that require merging of files for example, the pipeline can be supplied with a tabular separated value (TSV) file to enable such complex data handling.
```
Comment (jfy133) [inline content/05.methods.md]:
```suggestion
The pipeline can be started using (raw) FASTQ from sequencing or pre-mapped BAM files. Additionally, the pipeline requires a FASTA reference genome. If users have complex setups, e.g. multiple sequencing lanes that require merging of files for example, the pipeline can be supplied with a tabular separated value (TSV) file to enable such complex data handling.
```

File: content/05.methods.md [hunk 7]
```diff
@@ -2,30 +2,43 @@
...
+
+#### Monitoring
+
+Users can either monitor their pipeline execution on the commandline or utilize projects such as [Nextflow Tower](https://tower.nf) to monitor their analysis pipeline during runtime.
```
Comment (jfy133) [inline content/05.methods.md]:
```suggestion
Users can either monitor their pipeline execution on the using  nextflow's console running messages or utilize projects such as [Nextflow Tower](https://tower.nf) to monitor their analysis pipeline during runtime.
```

File: content/05.methods.md [hunk 8]
```diff
@@ -2,30 +2,43 @@
...
+
+#### Output
+
+The pipeline produces several dozen output files in various file formats, with a more detailed listing available in the user documentation. This includes metrics, statistical analysis data and standardized output files (BAM, VCF) for close inspection and further downstream analysis, as well as a MultiQC report. The latter can even be emailed to users automatically, when starting the pipeline with a dedicated `--email you@yourdomain.org`.
```
Comment (jfy133) [inline content/05.methods.md]:
```suggestion
The pipeline produces several dozen output files in various file formats, with a more detailed listing available in the user documentation. This includes metrics, statistical analysis data and standardized output files (BAM, VCF) for close inspection and further downstream analysis, as well as a MultiQC report. If a emailing daemon is setup on the server, the latter can even be emailed to users automatically, when starting the pipeline with a dedicated `--email you@yourdomain.org`.
```

File: content/05.methods.md [hunk 9]
```diff
@@ -2,30 +2,43 @@
...
 
 ## Data and software availability
 
 All code is available on github at
-[https://github.com/nf-core/eager](https://github.com/nf-core/eager) and 
-archived with Zenodo under the 
-DOI [10.5281/zenodo.1465061](https://doi.org/10.5281/zenodo.1465061). All
-test data is from the ENA public repository.
\ No newline at end of file
+[https://github.com/nf-core/eager](https://github.com/nf-core/eager) and archived with Zenodo under the DOI [10.5281/zenodo.1465061](https://doi.org/10.5281/zenodo.1465061). All test data is from the ENA public repository available under ENA IDs: FIXME
```
Comment (jfy133) [inline content/05.methods.md]:
```suggestion
[https://github.com/nf-core/eager](https://github.com/nf-core/eager) and archived with Zenodo under the DOI [10.5281/zenodo.1465061](https://doi.org/10.5281/zenodo.1465061). All test data is from the ENA public repository available under ENA IDs: **FIXME**
```

File: content/04-conclusion.md [hunk 10]
```diff
@@ -1,10 +1,8 @@
...
 will improve reproducibility and inclusion of rapidly increasing ancient DNA
 datasets, for both large and small laboratories. Extensive documentation also
-allow newcomers to the field get a practical understanding on how to interpret
-ancient DNA in the context of NGS data processing. Finally nf-core/eager brings
-in the latest tools and routine screening analysis commonly used in the field,
-and sets up the pipeline staying at the forefront of palaeogenetic analysis. 
+enables newcomers to the field get a practical understanding on how to interpret
+ancient DNA in the context of NGS data processing. Ultimately, nf-core/eager provides easier access to the latest tools and routine screening analysis commonly used in the field, and sets up the pipeline staying at the forefront of palaeogenetic analysis.
```
Comment (jfy133) [inline content/04-conclusion.md]:
```suggestion
ancient DNA in the context of NGS data processing. Ultimately, nf-core/eager provides easier access to the latest tools and routine screening analysis commonly used in the field, and sets up the for pipeline staying at the forefront of palaeogenetic analysis.
```

**Review by jfy133 — CHANGES_REQUESTED** [review]

File: content/03.results-and-discussion.md (hunk 1 above)
Comment (apeltzer) [inline content/03.results-and-discussion.md]:
```suggestion
estimates of the biological sex of a given individual with Sex.DetEERRmine
```

File: content/04-conclusion.md (hunk 3 above)
Comment (apeltzer) [reply content/04-conclusion.md:4]: Nextflow it should be across the entire document. nf-core too :-)

**Review by jfy133 — APPROVED** [review]
A few outstanding typos, but I'll fix these when doing another whole pass.

**Review by jfy133 — APPROVED** [review]

**Pull request merged**